runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -165,37 +165,36 @@ public void setEpoch(long epoch){
      * Whether or not this router is shutdown.
      */
     public volatile boolean shutdown;
-    /**
-     * The host that this router is routing requests for.
-     */
-    @Getter
-    String host;
-    /**
-     * The port that this router is routing requests for.
-     */
+
     @Getter
-    Integer port;
+    final NodeLocator node;
+
+    private final CorfuRuntimeParameters parameters;
+
+    @Deprecated
+    @Override
+    public Integer getPort() {
+        return node.getPort();
+    }
+
+    @Deprecated
+    public String getHost() {
+        return node.getHost();
+    }
+
     /**
      * Flag, if we are connected.
      */
     @Getter
     volatile Boolean connected;
 
-    private Boolean tlsEnabled = false;
-
     private SslContext sslContext;
-
-    private Boolean saslPlainTextEnabled = false;
-
-    private String saslPlainTextUsernameFile;
-
-    private String saslPlainTextPasswordFile;
-
     /**
      * Creates a new NettyClientRouter connected to the specified endpoint.
      *
      * @param endpoint Endpoint to connect to.
      */
+    @Deprecated
     public NettyClientRouter(String endpoint) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)